schema/observability/logs/logs.mapping

```diff
@@ -219,8 +219,11 @@
     }
   },
   "composed_of": [
-    "http_template",
+    "aws_elb_template",
```

Comment:
Per changes in the codebase, we want to start removing the `_template` and referring directly by name. We dynamically generate template names at runtime to support multiple versions and avoid collisions. The current integrations both have `composed_of` without `_template`.

Reply:
Details: #29

Reply:
this is not the actual file to use directly but the template for API to be used with the specific naming each user would select for his own needs.
For example a user may choose a prefix for all the index & component templates and this mapping template must respect this prefix configuration

Reply:
I will replace the fields with a template place holder to make it explicit